tests/e2e/testfiles/minimal-testfiles/storage_csm_powerscale.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: storage.dell.com/v1
+kind: ContainerStorageModule
+metadata:
+  name: isilon
+  namespace: isilon
+spec:
+  driver:
+    csiDriverType: "isilon"
+    configVersion: v2.12.0
+    forceRemoveDriver: true
+    common:
+      image: "dellemc/csi-isilon:nightly"
```

Comment:
Please add a TODO here to replace the nigthly image once the actual version is released

Reply:
I am fine with that but wouldn't it be a overhead for us to have it updated in each release so if it will be nightly it will get tested with all latest changes.

Reply:
Have shared the thread on slack regarding this query.